sdk/core/src/services/anchoringHandler.ts

```diff
@@ -0,0 +1,60 @@
+import { ELEM_DID_METHOD, JOLO_DID_METHOD } from '../_defaultConfig'
+import { RegistryApiService } from '@affinidi/internal-api-clients'
+import { DidDocumentService, KeysService } from '@affinidi/common'
+
+const getAnchoringParams = async (
```

Comment:
The name of this method is misleading; it does not just return anchoring params, but it also stores didDocument in IPFS.